<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>606</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>484</width>
    <height>606</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>patchgen GUI ver. 0.2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Patch file</string>
    </property>
    <widget class="QCheckBox" name="checkBoxExecutable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>executable</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPatchFile">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Output patchfile location:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPatchFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Sources</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Older version directory:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSrcDirectory">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSrcDirectory">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDestDirectory">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Newer version directory:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDestDirectory">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonCreate">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS UI Gothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonPatchFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browsePatchFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSrcDirectory</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseSrcDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDestDirectory</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseDestDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCreate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createPatchFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>browsePatchFile()</slot>
  <slot>browseSrcDirectory()</slot>
  <slot>browseDestDirectory()</slot>
  <slot>createPatchFile()</slot>
 </slots>
</ui>
